Client/mods/deathmatch/logic/CClientDisplayManager.cpp

```diff
@@ -88,7 +88,7 @@ void CClientDisplayManager::DoPulse()
     while (iter != m_List.end())
     {
         CClientDisplay* pObject = *iter;
-        if (pObject->IsExpired())
+        if (pObject->GetType() == DISPLAY_TEXT && pObject->IsExpired())
```

Comment:
Why did you change this code?

Reply:
This was for testing, I forgot to revert the change.